Errors with Django 1.10

I get the following when I added Django 1.10a1 to my tox environment:

```
.tox/py27-django110/lib/python2.7/site-packages/rest_framework_swagger/urls.py:2: in <module>
    from rest_framework_swagger.views import SwaggerResourcesView, SwaggerApiView, SwaggerUIView
.tox/py27-django110/lib/python2.7/site-packages/rest_framework_swagger/views.py:8: in <module>
    from django.shortcuts import render_to_response, RequestContext
E   ImportError: cannot import name RequestContext
```

I believe this is because `RequestContext` is supposed to come from `django.template`, not `django.shortcuts`.

https://docs.djangoproject.com/en/1.9/ref/templates/api/#django.template.RequestContext

AFAIK, changing this should be fully backward compatible.

--- a/rest_framework_swagger/views.py
+++ b/rest_framework_swagger/views.py
@@ -5,7 +5,8 @@ from django.conf import settings
 from django.views.generic import View
 from django.utils.safestring import mark_safe
 from django.utils.encoding import smart_text
-from django.shortcuts import render_to_response, RequestContext
+from django.shortcuts import render_to_response
+from django.template import RequestContext
 from django.core.exceptions import PermissionDenied
 from .compat import import_string
 

